### ISSUE TITLE
geograbber_process: object 'extent_name' not found

### ISSUE THREAD

**Issue by ranghetti** [ranghetti/sen2r#72]
Launching 
```r
sen2r:::geograbber_process("/path/to/SAFE/prods", "/path/to/out/prods", mask_type = "nodata")
```
 will result in the following error:
```
[2018-04-06 13:06:58] Starting sen2r execution.
[2018-04-06 13:06:59] Computing output names...
Error in extent_name %in% pm$extent_name : object 'extent_name' not found
```

**Comment by aceresi** [comment]
Dear Gigi, 

1. yes; moving the previously-calculated BOA file to another folder resulted in another kind of error: Error in prod_type %in% list_prods : object 'prod_type' not found

2. yes and no; the symptoms have changed -- now I get:
Error in get(fun, mode = "function", envir = parent.frame()) : 
  object 'sen2r' of mode 'function' was not found

**Comment by ranghetti** [comment]
I think second error occurs when the function is called before loading the library; 
```r
library(sen2r)
sen2r:::geograbber_process(...)
```
should fox.
Now I check the first one.

**Comment by ranghetti** [comment]
Fixed with 7bd679ac59c1c3c6f59406dc7edf8d7010b74537
